core/rawdb/accessors_chain.go

```diff
@@ -332,6 +336,18 @@ func WriteRawTransactions(db ethdb.RwTx, txs [][]byte, baseTxId uint64) error {
 	return nil
 }
 
+func DeleteTransactions(db ethdb.RwTx, bucket string, baseTxId uint64, amount uint32) error {
```

Comment:
This function needs to decrement the sequence for `TxId`, right?

Reply:
It does. It uses db.Put.